.github/workflows/docs.yml

```diff
@@ -0,0 +1,53 @@
+# https://github.com/mitmproxy/pdoc/blob/main/.github/workflows/docs.yml
+
+name: PyHa Documentation
+
+# build the documentation whenever there are new commits on main
+on:
+  push:
+    branches:
+      - main
+      - np-generate-docs
+
+# security: restrict permissions for CI jobs.
+permissions:
+  contents: read
+
+jobs:
+  # Build the documentation and upload the static HTML files as an artifact.
+  build:
+    runs-on: windows-latest
```

Comment:
Generally, we would prefer jobs run on `ubuntu-latest` unless you absolutely need Windows.

Reply:
I used ubuntu-latest but I couldn't figure out this error
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  random_device could not be read
/home/runner/work/_temp/aa96a6f4-a556-45f5-a8b1-05d0770bba35.sh: line 1:  1882 Aborted                 (core dumped) poetry run python -m pdoc ./PyHa -o docs/
Error: Process completed with exit code 134.
```
I suspect it has something to do with [this issue](https://github.com/pytorch/pytorch/issues/101152) but I figured generating docs worked on my machine (windows) so it would work on the action

Reply:
Please add an issue to PyHa referencing that issue then resolve this conversation.